installer/main.sh

```diff
@@ -91,11 +91,11 @@ if [ ! -z "$DESIRED_KOPS_CLUSTER_NAME" ] && [ -z "$SKIP_K8S_INSTALLATION" ]; the
 fi
 
 cd ingress
-. install.sh $RELEASE_PREFIX
+. install.sh $IMM_RELEASE_PREFIX
```

Comment:
If you exported this env, you could stop passing it over every component installer script :)